docs/curation-review-tut.md

```diff
@@ -0,0 +1,570 @@
+</p>
+
+<p align="center">
+  <b> Figure:</b> Login options
+</p>
+
+<br/>
+
+The cellxgene data portal offers multiple login options including sign-in with your gsuite and github accounts, or via email address.
+
+<br/>
+
+
+<!--- ![image](https://user-images.githubusercontent.com/25663501/113530303-a96a9e00-9593-11eb-91b7-0898d521314a.png) --->
+
+<p align="center">
+  <img src="https://user-images.githubusercontent.com/25663501/113530303-a96a9e00-9593-11eb-91b7-0898d521314a.png">
+</p>
+
+<p align="center">
+  <b> Figure:</b> Navigate to "My Collections"
+</p>
+
+<br/>
+
+It should be noted that collections are displayed with specific metadata about the collections (including assays(s) used, organism(s) profiled, number of cells in the collection, the tissue profiled, and the disease state studied). After you have logged in,  you will be able to navigate to the 'My Collections' page where you can view all of the datasets that you have uploaded to the portal (if you are just starting out, the 'My Collections' view will be empty).
+
+
+
+<!--- ![image](https://user-images.githubusercontent.com/25663501/113528943-18de8e80-9590-11eb-92ab-2a606879b70c.png) --->
+
+<!--- ![image](https://user-images.githubusercontent.com/25663501/113530607-6d840880-9594-11eb-8af7-17f75d3a8a40.png) --->
+
+
+<p align="center">
+  <img src="https://user-images.githubusercontent.com/25663501/113530607-6d840880-9594-11eb-8af7-17f75d3a8a40.png">
+</p>
+
+<p align="center">
+  <b> Figure:</b> My Collections
+</p>
+
+<br/>
+
+In the image above, we see an example of how the 'My Collections' page will look after you have uploaded a few datasets to the dataportal. Note that some of the collections in this example are published (publicly available) and one is private. The private collection is only available to people who you share the private link with. This allows for control over who views your datasets and for making amendmendents to datasets that have been uploaded to the portal previously. To add a new collection, we can simply click on the button highlighted above. 
+
+
+### Create a private collection
+
+<!--- ![image](https://user-images.githubusercontent.com/25663501/113528984-36135d00-9590-11eb-82a5-68ae4e5ddb1b.png) --->
+
+<p align="center">
+  <img width="500" src="https://user-images.githubusercontent.com/25663501/113528984-36135d00-9590-11eb-82a5-68ae4e5ddb1b.png">
+</p>
+
+<p align="center">
+  <b> Figure:</b> Creating a collection
+</p>
+
+<br/>
+
+Once you have clicked the 'Create Collection' button, you will be prompted to enter the following information:
+
+- Collection Name (name of the publication)
+- Description (abstract of publication)
+- Contact Name (PI name) and contact email (PI email)
+- Add Link (link to relevant resources such as the publication itself, lab website, consortia/department website, entry in GEO or other raw data sources, etc.)
```

Comment:
`Data Source` is not the same as `Raw Data`. It is intended to link back to the related page on the publisher portal such as BICCN.

Reply:
how are we defining publisher portal here?

Reply:
BICCN has their own portal. `Data Source` allows them to include a link back to their own related pages that may have more information, links to other visualization applications, etc.